### PR TITLE
Bugfix/caldav plugin strpos null

### PR DESCRIPTION
**Pull request by DeepDiver1975**

### CODE REVIEW

**Comment by staabm** [comment]
Does it require a 3.x backport?

**Comment by DeepDiver1975** [comment]
> Does it require a 3.x backport?

I don't think it's required because 3.x does not use strict types. Never the less backporting will not hurt

**Comment by DeepDiver1975** [comment]
Looks like I need to adjust some tests .....

**Comment by staabm** [comment]
> Looks like I need to adjust some tests .....

hmm travis passes, no?

**Comment by DeepDiver1975** [comment]
> hmm travis passes, no?

travis is fine - but the coderage dropped ....

**Comment by DeepDiver1975** [comment]
@staabm coverage is fine now. Merge? THX

File: lib/DAV/Sharing/Plugin.php [hunk 1]
```diff
@@ -172,7 +172,7 @@ public function httpPost(RequestInterface $request, ResponseInterface $response)
         $contentType = $request->getHeader('Content-Type');
 
         // We're only interested in the davsharing content type.
-        if (false === strpos($contentType, 'application/davsharing+xml')) {
+        if (false === strpos((string) $contentType, 'application/davsharing+xml')) {
```
Comment (staabm) [inline lib/DAV/Sharing/Plugin.php]: why do we need the string cast here, but not in the other cases?

Comment (staabm) [reply lib/DAV/Sharing/Plugin.php]: Since all other places touched with this PR use a explicit null?check I would prefer having one here, too

**Comment by DeepDiver1975** [comment]
@staabm ready now - THX

**Comment by staabm** [comment]
Thx

**Pull request merged**